docs/source/handling_advanced_stypes/handle_text.rst

```diff
@@ -1,38 +1,54 @@
 Handling Text Columns
-================================
+=====================
 
-:pyf:`PyTorch Frame` supports text columns by utilizing external pre-trained
+:pyf:`PyTorch Frame` supports text columns by utilizing or finetuning external pre-trained
 text embedding models, such as language models. Currently, we support
-:class:`stype.text_embedded<torch_frame.stype>` for text columns,
-which use a text embedding model to pre-encode text columns into embeddings
-(frozen during model training). We plan to support fine-tuning of the text embedding
-models soon.
+:class:`stype.text_embedded<torch_frame.stype>` and :class:`stype.text_tokenized<torch_frame.stype>`
+for text columns.
+:class:`stype.text_embedded<torch_frame.stype>` uses text embedding model(s) to pre-encode text columns into embeddings
+(those models are frozen during model training).
+:class:`stype.text_tokenized<torch_frame.stype>` finetunes underlying text embedding models by using the
+gradients backpropagated from the tabular learning. Now :pyf:`PyTorch Frame` pipeline supports supervised finetuning
+on text models with other libraries such as the `PEFT <https://huggingface.co/docs/peft/>`_.
```

Comment:
"Now ..." sounds a bit awkward to me. Can you rewrite to just say `text_tokenized` supports finetuning with peft?